Can't resolve '../node_modules/maplibre-gl/dist/maplibre-gl.css'

When I use the latest version of plotly.js 2.35.1, the following error occurs, but there was no such error with the previous version.
<img width="764" alt="iShot_2024-09-10_22 27 26" src="https://github.com/user-attachments/assets/8cad6ec0-05b0-441d-b484-8a03d7b057b3">

When attempting to upgrade from 2.35.0 to 2.35.1, I also experience this issue.

```
ERROR in ./node_modules/plotly.js/src/registry.js 277:8-67
Module not found: Error: Can't resolve '../node_modules/maplibre-gl/dist/maplibre-gl.css' in '/home/runner/work/cps-staff-portal/cps-staff-portal/node_modules/plotly.js/src'
resolve '../node_modules/maplibre-gl/dist/maplibre-gl.css' in '/home/runner/work/cps-staff-portal/cps-staff-portal/node_modules/plotly.js/src'
  using description file: /home/runner/work/cps-staff-portal/cps-staff-portal/node_modules/plotly.js/package.json (relative path: ./src)
    Field 'browser' doesn't contain a valid alias configuration
    using description file: /home/runner/work/cps-staff-portal/cps-staff-portal/node_modules/plotly.js/package.json (relative path: ./node_modules/maplibre-gl/dist/maplibre-gl.css)
      Field 'browser' doesn't contain a valid alias configuration
      /home/runner/work/cps-staff-portal/cps-staff-portal/node_modules/plotly.js/node_modules/maplibre-gl/dist/maplibre-gl.css doesn't exist
      .webpack.js
        Field 'browser' doesn't contain a valid alias configuration
        /home/runner/work/cps-staff-portal/cps-staff-portal/node_modules/plotly.js/node_modules/maplibre-gl/dist/maplibre-gl.css.webpack.js doesn't exist
      .web.js
        Field 'browser' doesn't contain a valid alias configuration
        /home/runner/work/cps-staff-portal/cps-staff-portal/node_modules/plotly.js/node_modules/maplibre-gl/dist/maplibre-gl.css.web.js doesn't exist
      .mjs
        Field 'browser' doesn't contain a valid alias configuration
        /home/runner/work/cps-staff-portal/cps-staff-portal/node_modules/plotly.js/node_modules/maplibre-gl/dist/maplibre-gl.css.mjs doesn't exist
      .js
        Field 'browser' doesn't contain a valid alias configuration
        /home/runner/work/cps-staff-portal/cps-staff-portal/node_modules/plotly.js/node_modules/maplibre-gl/dist/maplibre-gl.css.js doesn't exist
      .mts
        Field 'browser' doesn't contain a valid alias configuration
        /home/runner/work/cps-staff-portal/cps-staff-portal/node_modules/plotly.js/node_modules/maplibre-gl/dist/maplibre-gl.css.mts doesn't exist
      .ts
        Field 'browser' doesn't contain a valid alias configuration
        /home/runner/work/cps-staff-portal/cps-staff-portal/node_modules/plotly.js/node_modules/maplibre-gl/dist/maplibre-gl.css.ts doesn't exist
      .jsx
        Field 'browser' doesn't contain a valid alias configuration
        /home/runner/work/cps-staff-portal/cps-staff-portal/node_modules/plotly.js/node_modules/maplibre-gl/dist/maplibre-gl.css.jsx doesn't exist
      .tsx
        Field 'browser' doesn't contain a valid alias configuration
        /home/runner/work/cps-staff-portal/cps-staff-portal/node_modules/plotly.js/node_modules/maplibre-gl/dist/maplibre-gl.css.tsx doesn't exist
      as directory
        /home/runner/work/cps-staff-portal/cps-staff-portal/node_modules/plotly.js/node_modules/maplibre-gl/dist/maplibre-gl.css doesn't exist
 @ ./node_modules/plotly.js/src/core.js 12:15-36
 @ ./node_modules/plotly.js/lib/core.js 3:0-39
 @ ./resources/assets/js/components/reports-v2/graphs/BarChart.jsx 29:0-40 92:6-20 111:8-22 302:6-21
 @ ./resources/assets/js/components/reports-v2/snapshots/community-catalog/CommunityCatalog.jsx 15:0-45 47:53-61 59:55-63
 @ ./resources/assets/js/components/reports-v2/reports/Reports.constants.ts 8:43-101
 @ ./resources/assets/js/components/reports-v2/reports/Reports.tsx 58:28-58
 @ ./resources/assets/js/app.js 102:0-49

webpack 5.94.0 compiled with 1 error in 267682 ms
```

--- a/draftlogs/7146_fix.md
+++ b/draftlogs/7146_fix.md
@@ -1,0 +1,1 @@
+- Fix require path to maplibre-gl.css [[#7146](https://github.com/plotly/plotly.js/pull/7146)]

--- a/src/registry.js
+++ b/src/registry.js
@@ -274,7 +274,7 @@ function registerTraceModule(_module) {
 
     // add maplibre-gl CSS here to avoid console warning on instantiation
     if(bpmName === 'map') {
-        require('../node_modules/maplibre-gl/dist/maplibre-gl.css');
+        require('maplibre-gl/dist/maplibre-gl.css');
     }
 
     // if `plotly-geo-assets.js` is not included,

